package.json

```diff
@@ -36,7 +36,7 @@
   "scripts": {
     "ejslint": "ejslint generators/**/*.ejs && ejslint generators/**/*.ejs -d '&'",
     "lint": "eslint .",
-    "lint-fix": "npm run lint -- --fix",
+    "lint-fix": "npm run ejslint && npm run lint -- --fix",
     "mocha": "mocha generators --no-insight --forbid-only",
     "prettier-format": "prettier --write \"{,**/}*.{js,cjs,mjs,json,md,yml}\"",
     "prettier:check": "prettier --check \"{,src/**/}*.{md,json,yml,html,js,ts,tsx,css,scss,vue,java}\"",
```

Comment:
Will these include `.github/**`? I found that I had to add this to my prettier commands to get workflow files formatted. https://github.com/oktadev/auth0-full-stack-java-example/blob/main/package.json#L65

Reply:
`src/` should be dropped from the path pattern.
See https://github.com/jhipster/generator-jhipster-native/pull/29